CMakeLists.txt

```diff
@@ -209,11 +226,18 @@ endif()
 #else()
     add_library(SDL2main STATIC src/SDLmain/dummy/SDL_dummy_main.c)
 #endif()
-#add_library(SDL2::SDL2ain ALIAS SDL2main)
+add_library(SDL2::SDL2main ALIAS SDL2main)
 target_include_directories(SDL2main PRIVATE "include/SDL2")
+if(MINGW OR CYGWIN)
+  if(CMAKE_SIZEOF_VOID_P EQUAL 4)
```

Comment:
I wonder whether the `_` prefix and `@16` stdcall suffix applies to 32 bit targets other than x86, e.g. 32 bit arm...